src/redata.js

```diff
@@ -1,17 +1,19 @@
 /**
- * redata's data object definition
+ * Redata's data object definition.
+ *
  * @typedef {Object} Data
```

Comment:
I think you are referencing this typedef later in another block in lowercase, which is inconsistent.  Regarding upper or lower, I would go for lower but I don't have a strong opinion on this, so either is fine by me as long as it's consistent.